go/chat/flipmanager.go

```diff
@@ -129,6 +137,7 @@ func NewFlipManager(g *globals.Context, ri func() chat1.RemoteInterface) *FlipMa
 		dirtyGames:   make(map[string]chat1.FlipGameID),
 		forceCh:      make(chan struct{}, 10),
 		loadGameCh:   make(chan loadGameJob, 100),
+		visualizer:   NewFlipVisualizer(128, 50),
```

Comment:
this should be 256,100. Typically you want to generate 2x the pixels so it looks good in retina. the bottom of the 2 images I sent you in chat was 256,100 and the top was 128,50.